tests/src/whisk/core/controller/test/PackagesApiTests.scala

```diff
@@ -568,4 +569,54 @@ class PackagesApiTests extends ControllerTestCommon with WhiskPackagesApi {
             status should be(NotFound)
         }
     }
+
+    /*
+    it should "reject bind to non-package" in {
```

Comment:
These tests were pulled from a commit that built on additional changes not in this pr - rather than delete it, I just commented it out so I don't miss it when I rebase other commits. Same for other test that's commented out.

In this case, the test is checking that the response on the API call is Messages.requestedBindingIsNotValid - which requires changes to the API handler. The other tests assumes support for rules with actions in packages.